runtime/vm/createramclass.cpp

```diff
@@ -3018,7 +3018,7 @@ internalCreateRAMClassFromROMClassImpl(J9VMThread *vmThread, J9ClassLoader *clas
 			 *                 + J9ClassIsValueBased
 			 *                + J9ClassHasIdentity (inherited)
 			 *
-			 *              + Unused
+			 *              + J9ClassEnsureHashed
```

Comment:
The flag is inherited, so please change this comment to `J9ClassEnsureHashed (inherited)`